site/content/en/docs/tutorials/ai-playground.md

```diff
@@ -210,6 +210,7 @@ spec:
           --threads, "6",
           --no-warmup,
           --log-colors,
+          auto,
```

Comment:
I'm not sure auto is needed (since they have a default value), but maybe the behavior changed since we tested it. However all other options use `--key, value,` style, let's keep this style.